src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    addBook: () => {
+
+    },
+    removeBook: () => {
+
+    },
```

Comment:
- [ ] Please make sure to complete the `addBook` and `removeBook` reducers. These functions will require the state and action as parameters. The `addBook` function is responsible for adding a book to the `books` array in the state, while the `removeBook` function will remove a book from the `books` array based on its `id`.